### PR TITLE
Text styling fixes

### PR DESCRIPTION
**Pull request by estherhersh**

### CODE REVIEW

File: src/client/app/styles/sass/textEditor.scss [hunk 1]
```diff
@@ -73,3 +73,22 @@
   margin-right: -45px;
   margin-top: -5px;
 }
+
+h1{
```
Comment (MathuraMG) [inline src/client/app/styles/sass/textEditor.scss]: If these are for all h1 and h2 can we have them in base?

File: src/client/app/styles/sass/modals.scss [hunk 2]
```diff
@@ -20,20 +20,38 @@
...
+.pages_list ol li a{
+  color: $gray;
+}
+
+.file-modal__list ol{
+  list-style-position: inside;
+  -webkit-margin-before: 0;
+  -webkit-padding-start: 0px;
+}
+.file-modal__list ol li{
```
Comment (MathuraMG) [inline src/client/app/styles/sass/modals.scss]: can we put the classname instead of ol li?

File: src/client/app/styles/sass/base.scss [hunk 3]
```diff
@@ -1,10 +1,11 @@
-body, html, h1, h2, h3{
+body, html{
   margin: 0;
   padding: 0;
   font-family: 'Roboto', sans-serif;;
   background-color: $canvas-gray;
 }
 
+
```
Comment (MathuraMG) [inline src/client/app/styles/sass/base.scss]: can we remove this extra line?

File: src/client/app/styles/sass/textEditor.scss (hunk 1 above)
Comment (estherhersh) [reply src/client/app/styles/sass/textEditor.scss]: I guess they would be for all h1s and h2s but we don't use it anywhere else and I styled them for the text editor dropdown....should I still move it?

File: src/client/app/styles/sass/modals.scss (hunk 2 above)
Comment (estherhersh) [reply src/client/app/styles/sass/modals.scss]: The classnames also apply to the links in the dropdown. I'm having trouble targeting the list in the modal without messing up the dropdown list. Would you mind taking a look at it?

**Pull request merged**